File: old_src/main/java/dev/vini2003/hammer/core/registry/server/HCEvents.java
package dev.vini2003.hammer.core.registry.server;

import dev.vini2003.hammer.core.api.client.util.InstanceUtil;
import net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents;
import net.fabricmc.loader.api.FabricLoader;
import net.minecraft.server.MinecraftServer;

public class HCEvents {
	public static void init() {
		ServerLifecycleEvents.SERVER_STARTED.register((server) -> {
			InstanceUtil.setServerSupplier(() -> (MinecraftServer) FabricLoader.getInstance().getGameInstance());
		});
	}
}

File: hammer-chat/src/main/kotlin/dev/vini2003/hammer/chat/registry/client/HCEvents.kt
package dev.vini2003.hammer.chat.registry.client

import dev.vini2003.hammer.chat.client.listener.RenderHudWarningsCallback
import net.fabricmc.fabric.api.client.rendering.v1.HudRenderCallback

object HCEvents {
	fun init() {
		HudRenderCallback.EVENT.register(RenderHudWarningsCallback)
	}
}